packages/docusaurus-theme-classic/src/theme/DocSidebar/index.js

```diff
@@ -171,19 +171,16 @@ function DocSidebar(props) {
           )}
         </button>
         <ul className="menu__list">
-          {sidebarData.map(
-            item =>
-              item.items.length > 0 && (
```

Comment:
Do we not need to check anymore?

Reply:
if the first item is not a category, items is undefined. Checking is alrd done in the DocSidebarItem itself

Reply:
without this loc removal, it will error